src/Components/Components/src/RenderTree/ArrayBuilderSegment.cs

```diff
@@ -7,10 +7,12 @@
 
 namespace Microsoft.AspNetCore.Components.RenderTree
 {
+
     /// <summary>
-    /// Represents a range of elements within an instance of <see cref="ArrayBuilder{T}"/>.
+    /// Types in the Microsoft.AspNetCore.Components.RenderTree are not recommended for use outside
+    /// of the Blazor framework. These types will change in a future major release.
     /// </summary>
-    /// <typeparam name="T">The type of the elements in the array</typeparam>
+    /// <typeparam name="T"></typeparam>
```

Comment:
Was it intentional to change this line?

Reply:
Well I guess so, since you did the same below. Not sure why though.

Reply:
Yes. The point is to just make the documentation blank other than the stern warning about breaking changes. It's wierd to fill in the summary with boilerplate but then provide detail in the other thigns. 

In other cases, I preserved existing documentation *as comments* because it has some value to us.